app/jsx/components/TabContent1Comp.jsx

```diff
@@ -11,69 +11,108 @@ class TabContent1Comp extends React.Component {
     let p = this.props
     return (
       <div className="c-tabcontent">
-        <ItemActionsComp />
+        <ItemActionsComp status={p.status} content_type={p.content_type} id={p.id} />
```

Comment:
I think there's a cute shorthand for the three expressions above using ES7 destructuring syntax, but heck if I can figure it out. Good to go.